melodic/distribution.yaml

```diff
@@ -11435,6 +11435,12 @@ repositories:
       url: https://github.com/SteveMacenski/spatio_temporal_voxel_layer.git
       version: melodic-devel
     status: maintained
+  sr_hand_detector:
+    release:
```

Comment:
Would you like to add a source and/or doc entry to this?  It makes it much easier for users to find where the sources come from.

Reply:
Hi @clalancette Yes, but actually I just updated the version, so I will close this PR and create a new one with updated version and source. Sorry about the confusion.